components/draggable/README.md

```diff
@@ -6,6 +6,14 @@
 
 The component accepts the following props:
 
+### draggable
+
+If the component is in a draggable state or not. If false, the component is rendered but it is not possible to drag it. Defaults to true.
```

Comment:
"If the component is in a draggable state or not." is a bit confusing, I think the point of this is to set the "dragability" of the block, right? Would this be better as:

```markdown
### draggable

Allow the component to be dragged in the editor; defaults to `true`. If `false`, the component is rendered but it is not possible to drag the component.
```

Reply:
Agreed, it's very confusing that `Draggable` component can't be draggable in some circumstances. Ideally, it shouldn't be rendered at all when it is not draggable.